src/internationalization/dictionaries/pt.json

```
@@ -45,4 +46,23 @@
     "projects": "Projetos"
   },
   "social_media": "Redes sociais"
+  ,
+  "footer": {
+    "section1": {
+      "title": "CeSIUM",
+      "items": ["Notícias", "Sobre nós", "Equipa", "Departmentos"]
+    },
+    "section2": {
+      "title": "Atividades",
+      "items": ["Projetos", "Eventos", "Parcerias"]
+    },
+    "section3": {
+      "title": "Estudantes",
+      "items": ["Torna-te sócio", "Torna-te colaborador", "Anuário"] 
+    },
+    "section4": {
+      "title": "Links Úteis",
+      "items": ["Calendarium", "Blackboard", "Portal do Aluno", "Página de LEI"]
+    }
+  }
```

To simplify, maybe we could turn this into an array of sections and also include the link information of each item. This way, on the Footer code, we can just iterate through `dict.sections` with `map` and do what we need to do, wanna give it a try? 👀 

```suggestion
  "footer": {
    "sections": [{
        "title": "CeSIUM",
        "items": ["Notícias", "Sobre nós", "Equipa", "Departmentos"],
        "links": ["#", "#", "#", "#"]
      },
      {
        "title": "Atividades",
        "items": ["Projetos", "Eventos", "Parcerias"],
        "links": ["#", "#", "#", "#"]
      },
      {
        "title": "Estudantes",
        "items": ["Torna-te sócio", "Torna-te colaborador", "Anuário"],
        "links": ["#", "#", "#", "#"]
      },
      {
        "title": "Links Úteis",
        "items": ["Calendarium", "Blackboard", "Portal do Aluno", "Página de LEI"],
        "links": ["#", "#", "#", "#"]
      }]
  }
```